demo/config/routes.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
+# :nocov:
 Rails.application.routes.draw do
-  get '/view-components', to: redirect('/view-components/stories/')
-  get '/', to: redirect('/view-components/stories/')
+  if Rails.env.production?
+    get '/view-components', to: redirect('/view-components/stories/')
```

Comment:
Does this mean that this route is only available in prod now? I think we'd want to have it available in development still.

Reply:
The storybook server runs separately at :5000. This doesn't effect that